gallery/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "9.16.1",
+  "version": "9.17.0",
   "description": "Gallery application to demonstrate usage and look of Sonatype shared UI components",
   "main": "src/main.ts",
   "scripts": {
     "start": "npm-run-all lint webpack-dev-server",
     "build": "npm-run-all lint webpack copy-assets",
-    "copy-assets": "cpx \"{src/index.html,src/assets/images/favicon*}\" dist",
+    "copy-assets": "cpx \"{src/index.html,src/assets/images/favicon*}\" dist; cpx hosted-versions.json dist",
```

Comment:
We try to maintain platform independence in our buld scripts, so a semi-colon is a no-go.  Use `npm-run-all` instead

Reply:
> We try to maintain platform independence in our buld scripts, so a semi-colon is a no-go. Use npm-run-all instead

Or really I feel like this could just be combined into one `cpx` call by adding the new file to the existing glob

Reply:
> > We try to maintain platform independence in our buld scripts, so a semi-colon is a no-go. Use npm-run-all instead
> 
> Or really I feel like this could just be combined into one `cpx` call by adding the new file to the existing glob

Thanks! This is no longer needed and have been removed with the new implementation